sdk-api-src/content/cfapi/nf-cfapi-cfcreateplaceholders.md

```diff
@@ -57,6 +57,8 @@ Creates one or more new placeholder files or directories under a sync root tree.
 ### -param BaseDirectoryPath [in]
 
 Local directory path under which placeholders are created.
+Note that this represents the directory where the placeholder needs to be created. Hence it will be sync root only
```

Comment:
Note that this directory must be the immediate parent directory of the placeholders being created. For example, if the sync root of the provider is c:\SyncRoot and a placeholder is being created in a child directory of the sync root such as c:\SyncRoot\ChildDir, then BaseDirectoryPath = c:\SyncRoot\ChildDir.

Reply:
Done. Please refer the latest commit !